xforce.json

```diff
@@ -1,16 +1,16 @@
 {
     "appid": "653a4735-1eed-4cb9-98c4-b40408e60b22",
-    "name": "IBM XForce",
-    "description": "This app implements various 'investigative' actions on the 'IBM XForce' device",
+    "name": "IBM X-Force Exchange",
+    "description": "This app implements various 'investigative' actions on the 'IBM X-Force Exchange' device",
     "publisher": "AvantGarde Partners",
-    "package_name": "phantom_xforce",
+    "package_name": "phantom_ibmx-forceexchange",
```

Comment:
I plan to name the test suite repo to be `ibmx-forceexchange`.
The test playbook will also be named `phantom_ibmx-forceexchange`

Reply:
Do all our current test playbooks still use the "phantom_" prefix?